html/dist/voiceChat.css

```diff
@@ -1,17 +1,12 @@
-﻿
-* {
+﻿* {
     margin: 0 0;
     padding: 0 0;
     border-spacing: 0px;
     box-sizing: content-box;
 }
-@font-face {
-    font-family: "GTAV_font_6";
-    src: url('./font6.ttf') format("truetype");
-}
 
 #iframes_container {
-	//display: none;
+	display: none;

```

Comment:
https://github.com/rt-2/RageMP-SAARPV-voiceChat/blob/master/html/index.html#L107 is repeating it btw

Reply:
Fixed.